test/clj_ldap/test/client.clj

```diff
@@ -139,6 +140,18 @@
           :cn (-> person-a* :object :cn)
           :sn (-> person-a* :object :sn)})))
 
+(deftest some-bind
+  (is (= (ldap/bind *conn* (:dn person-a*) "passa")
+         {:code 0, :name "success"}))
+  (is (thrown? LDAPException (ldap/bind *conn* (:dn person-a*) "notthepass")))
+  (is (thrown? LDAPException (ldap/bind *conn* "cn=does,ou=not,cn=exist" "password"))))
+
+(deftest bad-bind?
+  (is (= (ldap/bind? *conn* (:dn person-a*) "not-the-password")
+         false))
```

Comment:
I believe `clojure.test/is` assumes the first parameter is the expected value and subsequent parameter(s) are the actual value(s). Test failure messages will make more sense if these test cases are written like:

```
(is (= false (ldap/bind? *conn* (:dn person-a*) "not-the-password")))
```

Reply:
Went through the test suite and correct.